rest-service/manager_rest/deployment_update/manager.py

```diff
@@ -165,7 +164,8 @@ def extract_steps_from_deployment_update(self, deployment_update_id):
             step_extractor.extract_steps(deployment_update)
 
         if not unsupported_steps:
-            deployment_update.steps = supported_steps
+            deployment_update.steps = [step.to_model_deployment_update_step()
```

Comment:
the conversion method should probably be in manager.py rather than a part of the object? consider